orchagent/fdborch.cpp

```diff
@@ -1410,7 +1416,9 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
             attrs.push_back(attr);
         }
     }
-
+    attr.id = SAI_FDB_ENTRY_ATTR_PACKET_ACTION;
```

Comment:
Should there be changes in fdborch to push the packet to PACD socket? https://github.com/sonic-net/SONiC/blob/master/doc/pac/Port%20Access%20Control.md#313-mab-packet-receive-flow. How are the packets delivered for authentication when MAB is enabled on the interface please?

Reply:
These packets that come to CPU are received by PACD as it registers socket for the same. If  MAB is enabled, then PACD informs , the information that is required for authentication,  such as mac address and the interface to MAB process. The communication between the PACD and MABD uses simple client-server socket communication.